src/sharedData/components/SharedDataCard.js

```diff
@@ -86,6 +86,9 @@ const SharedFilesCard = props => {
                   ext => `*.${ext}`
                 ).join(', '),
                 name: owner.name,
+                item: owner.defaultGroup
```

Comment:
Suggestion: `type` feels a little less generic than `item`, I don't think `item` is telling me anything about the data, maybe even `entityType`?